doc/py_tutorials/py_feature2d/py_brief/py_brief.markdown

```diff
@@ -61,18 +61,18 @@ img = cv2.imread('simple.jpg',0)
 star = cv2.xfeatures2d.StarDetector_create()
 
 # Initiate BRIEF extractor
-brief = cv2.BriefDescriptorExtractor_create()
+brief = cv2.xfeatures2d.BriefDescriptorExtractor_create()
 
 # find the keypoints with STAR
 kp = star.detect(img,None)
 
 # compute the descriptors with BRIEF
 kp, des = brief.compute(img, kp)
 
-print brief.getInt('bytes')
+print brief.getDescriptorSize()
```

Comment:
Here should be `brief.descriptorSize()` and there are several similar mistakes in this commit.